pkg/validate/secrets.go

```diff
@@ -61,14 +61,12 @@ func (s Credentials) ValidatePath(ctx context.Context) error {
 func (s Credentials) buildCredentialserences() map[string]build.BuildReason {
 	// Validate if the referenced secrets exist in the namespace
 	secretRefMap := map[string]build.BuildReason{}
-	if s.Build.Spec.Output.Credentials != nil && s.Build.Spec.Output.Credentials.Name != "" {
-		secretRefMap[s.Build.Spec.Output.Credentials.Name] = build.SpecOutputSecretRefNotFound
+	if s.Build.Spec.Output.PushSecret != nil && *s.Build.Spec.Output.PushSecret != "" {
```

Comment:
With omitEmpty, this will actually never be `""`?

Reply:
I don't think is due to the omitEmpty, but rather because the pointer is not nil(cannot have a non-empty string), I removed the check against ""